### PR TITLE
Don't run Fx chaos mode test verification step

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

**Comment by Hexcles** [comment]
Should this be default/opt-in?

**Comment by jgraham** [comment]
I don't know, but that's a much harder change to make since it involves changing gecko and waiting for it to sync. Which I can do, but I'm not convinced the overhead is worth it.

File: tools/ci/tc/tasks/test.yml [hunk 1]
```diff
@@ -230,6 +230,7 @@ tasks:
               --
               --channel=${vars.channel}
               --verify
+              --verify-no-chaos-mode
```
Comment (stephenmcgruer) [inline tools/ci/tc/tasks/test.yml:233]: Please make sure to document the reasoning for this either in the yml or at least the commit message, to give future-us an understanding of why we made this change :)

**Review by stephenmcgruer — APPROVED** [review]
If that is sufficiently painful, then lgtm.

**Pull request merged**